modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -85,7 +89,7 @@ When authentication is enabled, the Bearer token used to query the reporting API
 * metering-admin
 * metering-viewer
 
-The `metering-operator` is capable of creating `RoleBindings` for you, granting these permissions by specifying a list of subjects in the `spec.permissions` section. For an example, see the `advanced-auth.yaml` example configuration below.
+The `metering-operator` is capable of creating RoleBindings for you, granting these permissions by specifying a list of subjects in the `spec.permissions` section. For an example, see the `advanced-auth.yaml` example configuration below.
```

Comment:
Metering Operator or `metering-operator` resource - you need to qualify objects with their type.
never "below"
> ! below adj
Do not use to indicate a relative location in a document, as in “the information below.” Use “the information
later in this section.”
Alternatively, you can use “following” to refer to a table, to art, or to a list. Example: “The following table
shows ...”